src/lib/types.ts

```diff
@@ -51,7 +51,7 @@ export interface Column<T> {
     header?: React.ReactNode;
     headerTitle?: string;
     className?: string;
-    width?: number;
+    width?: number | string;
```

Comment:
Could you please update `width` type in README?

Reply:
Sure